hugegraph-core/src/main/java/com/baidu/hugegraph/backend/serializer/BinarySerializer.java

```diff
@@ -663,7 +701,7 @@ private Query writeQueryEdgePrefixCondition(ConditionQuery cq) {
 
             if (key == HugeKeys.OWNER_VERTEX ||
                 key == HugeKeys.OTHER_VERTEX) {
-                writePrefixQueryEdge((Id) value, buffer);
+                writePartitionedId((Id) value, buffer, HugeType.EDGE);//此处要加一个判断  预分区非开启 要关掉
```

Comment:
can remove the comment since add enablePartition in writePartitionedId() method?